### PR TITLE
[CELEBORN-746][BUILD] Rename project files from rss-xx to celeborn-xx

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Rename project files from rss-xx to celeborn-xx

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,8 +58,6 @@ reports/
 scalastyle-on-compile.generated.xml
 scalastyle-output.xml
 scalastyle.txt
-rss-*.tgz
-rss-*-bin*
 emr-*.tgz
 emr-*-bin*
```
Comment (pan3793) [inline .gitignore:62]: The `emr-*` could be deleted too?

Comment (AngersZhuuuu) [reply .gitignore:62]: Yea， also `celeborn-*.tgz`

**Review by pan3793 — APPROVED** [review]

**Comment by cfmcgrady** [comment]
as we are here, would you mind updating the terminology within the Drawio file and regenerating the jpg file?

**Comment by AngersZhuuuu** [comment]
> as we are here, would you mind updating the terminology within the Drawio file and regenerating the jpg file?

Done

File: .gitignore [hunk 2]
```diff
@@ -58,11 +58,6 @@ reports/
 scalastyle-on-compile.generated.xml
 scalastyle-output.xml
 scalastyle.txt
-rss-*.tgz
-rss-*-bin*
-emr-*.tgz
-emr-*-bin*
-celeborn-*.tgz
 apache-celeborn-*.tgz
 celeborn-*-bin*
```
Comment (pan3793) [inline .gitignore:62]:
```suggestion
```

**Comment by cfmcgrady** [comment]
> > as we are here, would you mind updating the terminology within the Drawio file and regenerating the jpg file?
> 
> Done

forget uploading the regenerated jpg file?

**Comment by AngersZhuuuu** [comment]
> > > as we are here, would you mind updating the terminology within the Drawio file and regenerating the jpg file?
> > 
> > 
> > Done
> 
> forget uploading the regenerated jpg file?

Not used now, after discuss, delete it directly

**Review by FMX — APPROVED** [review]
LGTM. Thanks.

**Review by cfmcgrady — APPROVED** [review]

**Pull request closed without merging**